internal/resources/ekscluster/resource_ekscluster.go

```diff
@@ -132,6 +140,7 @@ var configSchema = &schema.Schema{
 				Description: "The metadata to apply to the cluster to assist with categorization and organization",
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
+				Computed:    true,
```

Comment:
is this intended ?, User will not be able to provide values if marked as computed